README.md

```diff
@@ -17,6 +17,24 @@
 
 `pip install git+https://github.com/Tomme/dbt-athena.git`
 
+### Prerequisites
+
+To start, you will need an S3 bucket, for instance `my-staging-bucket` and an Athena database:
+
+```sql
+CREATE DATABASE IF NOT EXISTS analytics_dev
+COMMENT 'Analytics models generated by dbt (development)'
+LOCATION 's3://my-staging-bucket/'
+WITH DBPROPERTIES ('creator'='Foo Bar', 'email'='foo@bar.com');
+```
+
+Take note of your AWS region code (e.g. `us-west-2` or `eu-west-`).
```

Comment:
`eu-west-` - quick typo - I'm guessing it is meant to be `eu-west-1`?

Reply:
Thanks for catching this. Now resolved. ✅